apps/src/sites/studio/pages/devise/registrations/edit.js

```diff
@@ -49,6 +53,15 @@ $(document).ready(() => {
     );
   }
 
+  if (experiments.isEnabled(experiments.ACCOUNT_DELETION_NEW_FLOW)) {
+    const deleteAccountMountPoint = document.getElementById('delete-account');
+    // Replace deleteAccountMountPoint Rails contents with DeleteAccount component.
+    ReactDOM.render(
```

Comment:
I think this needs to check if the mount point actually exists - it's possible it won't be rendered if the user doesn't have permission to delete their own account.

Reply:
you're right! i'll add that check.